plugins/amazonq/codetransform/jetbrains-community/src/software/aws/toolkits/jetbrains/services/codemodernizer/panels/managers/BuildProgressSplitterPanelManager.kt

```diff
@@ -84,7 +84,7 @@ class BuildProgressSplitterPanelManager(private val project: Project) :
     }
 
     fun defaultProgressData() = listOf(
-        BuildProgressStepTreeItem(message("codemodernizer.toolwindow.progress.waiting"), BuildStepStatus.WORKING, ProgressStepId.ACCEPTED),
+        BuildProgressStepTreeItem(message("codemodernizer.toolwindow.progress.uploading"), BuildStepStatus.WORKING, ProgressStepId.UPLOADING),
```

Comment:
Are we still uploading at this point though? Is it not that the transformation hub is not even shown until the data has been uploaded properly?

Reply:
![image](https://github.com/aws/aws-toolkit-jetbrains/assets/60020664/296bae96-4971-45a6-bb46-5f960ff9db22)

Reply:
Transformation Hub shows up just before the upload URL is created